src/RamTransforms.h

```diff
@@ -476,13 +476,17 @@ class IndexedInequalityTransformer : public RamTransformer {
         return "IndexedInequalityTransformer";
     }
 
-    // converts a box query into a corresponding filter operation
+    // converts a box query into a corresponding partial box query operation
+    // currently this will turn every box query into a filter operation
```

Comment:
Use doxygen notation to comment the method. Something like:
```suggestion
    /** Converts a box query into a corresponding partial box query operation.
     *  This will turn every box query into a filter operation.
     */
```